board.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1 style="text-align: center;">Triqui</h1>
+    <br>
+    <table>
+        <tr style="color: red;">
```

Comment:
- what if the winner line is not in a row or a column? I suggest using a CSS class instead